app/assets/javascripts/components/locales/es.jsx

```diff
@@ -0,0 +1,49 @@
+const es = {
+  "column_back_button.label": "Atrás",
+  "lightbox.close": "Cerrar",
+  "loading_indicator.label": "Cargando...",
+  "status.mention": "Mencionar",
+  "status.delete": "Borrar",
+  "status.reply": "Responder",
+  "status.reblog": "Republicar",
+  "status.favourite": "Favorito",
+  "status.reblogged_by": "{name} republicado",
+  "video_player.toggle_sound": "Act/Desac. sonido",
+  "account.mention": "Mención",
+  "account.edit_profile": "Editar perfil",
+  "account.unblock": "Desbloquear",
+  "account.unfollow": "Dejar de seguir",
+  "account.block": "Bloquear",
+  "account.follow": "Seguir",
+  "account.block": "Bloquear",
+  "account.posts": "Publicaciones",
+  "account.follows": "Seguir",
+  "account.followers": "Seguidores",
+  "account.follows_you": "Te sigue",
+  "getting_started.heading": "Primeros pasos",
+  "getting_started.about_addressing": "Puedes seguir a gente si conoces su nombre de usuario y el dominio en el que están registrados, introduciendo algo similar a una dirección de correo electrónico en el formulario en la parte superior de la barra lateral.",
+  "getting_started.about_shortcuts": "Si el usuario que buscas está en el mismo dominio que tú, simplemente funcionará introduciendo el nombre de usuario. La misma regla se aplica para mencionar a usuarios.",
+  "getting_started.about_developer": "Puedes seguir al desarrollador de este proyecto en Gargron@mastodon.social",
+  "column.home": "Inicio",
+  "column.mentions": "Menciones",
+  "column.public": "Público",
+  "tabs_bar.compose": "Redactar",
+  "tabs_bar.home": "Inicio",
+  "tabs_bar.mentions": "Menciones",
+  "tabs_bar.public": "Público",
+  "compose_form.placeholder": "¿En qué estás pensando?",
+  "compose_form.publish": "Publicar",
+  "navigation_bar.settings": "Ajustes",
+  "navigation_bar.public_timeline": "Línea de tiempo pública",
```

Comment:
I have just one note. This text will be displayed next to the avatar on the sidebar, could you shorten it somehow so it doesn't break into two lines? The column.public (aka column header) could contain the longer version.